crates/bevy_ui/src/ui_node.rs

```diff
@@ -531,6 +724,37 @@ impl Default for JustifyContent {
     }
 }
 
+/// How this node should be aligned in the main axis
+#[derive(Copy, Clone, PartialEq, Eq, Debug, Serialize, Deserialize, Reflect)]
+#[reflect(PartialEq, Serialize, Deserialize)]
+pub enum JustifySelf {
+    Auto,
```

Comment:
Missing doc strings.

Reply:
I'm not quite sure what `JustifySelf` does. I thought it was a `grid` property but it's not part of the Taffy `grid` feature.

Reply:
It doesn't seem to do anything, I removed it for now.

Reply:
Ping @nicoburns

Reply:
> I'm not quite sure what JustifySelf does. I thought it was a grid property but it's not part of Taffy's grid feature.

Correct. It not being feature flagged by `grid` in Taffy is a bug (unnoticed because having an extra style property doesn't break anything).